sklearn/cluster/tests/test_affinity_propagation.py

```diff
@@ -29,11 +32,14 @@
     random_state=0,
 )
 
+# TODO: AffinityPropagation must preserve dtype for its fitted attributes
+# and test must be created accordingly to this new behavior.
```

Comment:
maybe add a link to the corresponding issue https://github.com/scikit-learn/scikit-learn/issues/11000